docs/getting-started/remote-data.md

```diff
@@ -54,7 +54,7 @@ Serving HTTP on 0.0.0.0 port 8000 ...
 
 ### Using Zarr.js to access the data
 ```javascript
-import { slice, openArray } from "../dist/zarr.es5.js";
+import { slice, openArray } from "zarr";
```

Comment:
Not sure if this needs the relative path. In the other `getting-started` sections we just import like we are using a bundler / node.